modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/upgrade/v3_9_0/NotificationRecipientSettingUpgradeProcess.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.notification.internal.upgrade.v3_9_0;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.dao.jdbc.AutoBatchPreparedStatementUtil;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.kernel.uuid.PortalUUID;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Timestamp;
+
+/**
+ * @author Selton Guedes
+ */
+public class NotificationRecipientSettingUpgradeProcess extends UpgradeProcess {
+
+	public NotificationRecipientSettingUpgradeProcess(PortalUUID portalUUID) {
+		_portalUUID = portalUUID;
+	}
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		try (PreparedStatement preparedStatement1 = connection.prepareStatement(
+				"select notificationRecipientId, companyId, userId, userName " +
+					"from NotificationRecipient");
+			PreparedStatement preparedStatement2 =
+				AutoBatchPreparedStatementUtil.concurrentAutoBatch(
+					connection,
+					StringBundler.concat(
+						"insert into NotificationRecipientSetting (uuid_, ",
+						"notificationRecipientSettingId, companyId, userId, ",
+						"userName, createDate, modifiedDate, ",
+						"notificationRecipientId, name, value) values (?, ?, ",
+						"?, ?, ?, ?, ?, ?, ?, ?)"));
+			ResultSet resultSet = preparedStatement1.executeQuery()) {
+
+			while (resultSet.next()) {
+				preparedStatement2.setString(1, _portalUUID.generate());
```

Comment:
Could you replace `_portalUUID.generate()` to `PortalUUIDUtil.generate()`?
What do you think?

Reply:
We talked and decided to keep the `_portalUUID.generate()`